changelog.dd

```diff
@@ -327,6 +329,15 @@ static assert( isFinal!(C.ff));
 -------
 )
 
+$(LI $(LNAME2 generate, `std.range.generate` fixed to be a proper range.)
```

Comment:
I would add a note here about the clarified rules with a link when https://github.com/dlang/phobos/pull/4511 is pulled